airbyte-integrations/connectors/destination-bigquery/src/test-integration/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQueryGcsRawOverrideDisableTypingDedupingTest.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.bigquery.typing_deduping;
+
+public class BigQueryGcsRawOverrideDisableTypingDedupingTest extends AbstractBigQueryTypingDedupingTest {
+
+  @Override
+  public String getConfigPath() {
+    return "secrets/credentials-1s1t-disabletd-gcs-raw-override.json";
+  }
+
+  @Override
+  protected String getRawDataset() {
+    return "overridden_raw_dataset";
+  }
+
+  @Override
+  protected boolean disableFinalTableComparison() {
+    return true;
+  }
+
+  @Override
```

Comment:
```suggestion
  @Override
  @Disabled
```

(same for the other empty test overrides)

Reply:
Yeah I was doing trial and error until you found I overrode in the wrong class. I'll put it back as `@Disabled`